data-raw/standard-format-data-prep/covariate_stock_recruit.Rmd

```diff
@@ -105,20 +105,98 @@ butte_std_temp <- butte_std_temp |>
 spawning_months <- c("08", "09", "10", "11", "12")
 
 butte_spawn_temps <- butte_std_temp |> 
-  filter(month %in% spawning_months) 
+  filter(month%in%spawning_months) |> 
+  filter(date!= as.Date("2000-12-09"))  #removes one day with NA max temp 
+
 #Making sure we only have the months we need 
 table(butte_spawn_temps$month)
 
+#Delete "month field" 
+butte_spawn_temps |> select(-month)
+
+#Calculating Butte 7DADM
```

Comment:
Great job paying attention to NAs. Makes sense that the first and last window would be NA. We could also experiment with using "extend" for the `fill` parameter.

Reply:
Something we need to make sure to do is group by site/subsite. I can explain this more.